src/pages/profile/index.tsx

```diff
@@ -6,16 +6,42 @@ import { useGitHubQuery } from "@/hooks";
 import Image from "next/image";
 import Link from "next/link";
 import { exportAsImage } from "@/utils";
+import GitHubCalendar from "react-github-calendar";
+import { useState } from "react";
+
+interface Activity {
+  date: string;
+  count: number;
+  level: 0 | 1 | 2 | 3 | 4;
+}
 
 export default function Profile() {
   const { data } = useGitHubQuery<UserProfile>(userProfileQuery);
+  const [checked, setChecked] = useState<boolean>(false);
```

Comment:
Can we use a more self explanatory variable here? `checked` is too generic

Reply:
Agreed.